docs/semgrep-cloud-platform/gitlab-mr-comments.md

```diff
@@ -88,14 +108,18 @@ Only rules in the **Comment** and **Block** columns of your [Policies](https://s
 
 <DisplayTaintedDataIntro />
 
-### Viewing the path of tainted data in MR comments
+### View the path of tainted data in MR comments
 
 To enable dataflow traces in your CI pipeline, fulfill the following prerequisites:
 
 :::info Prerequisites
-- Enable GitLab merge request Semgrep comments. For more details, see [Enabling GitLab merge request comments](#enabling-gitlab-merge-request-comments) section.
+- Set up GitLab merge request Semgrep comments, as described on this page.
```

Comment:
```suggestion
- Set up Semgrep to push GitLab merge request comments as described on this page.
```

Reply:
I'm going with "to post GitLab merge request comments..." for this one :)